blog/views.py

```diff
@@ -0,0 +1,102 @@
+from django.shortcuts import render
+from blog.models import Articles ,Tags, Article_Tag_Mapping, Comments
+from django.contrib import messages
+from user.models import Users
+from django.http import HttpResponse, HttpResponseRedirect
+import random
+import string
+
+# Create your views here.
+def create(request):
+    if request.method == "POST":
+        form = (request.POST)
```

Comment:
add the check for form validation. The call form.cleaned_data to get the cleaned form. Then save the form